src/pages/Downloads.js

```diff
@@ -1,22 +1,121 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import { Page } from '../components'
 import { Text } from '@aragon/ui'
 import styled from 'styled-components'
 import Section from '../components/Page/Section'
+import figmaSVG from './assets/figma.svg'
+import SketchSVG from './assets/sketch.svg'
+import FramerSVG from './assets/framer.svg'
+import DownloadIconSVG from './assets/downloadicon.svg'
 
 const Downloads = () => (
   <Page path="/downloads">
     <Section>
-      <Container />
+      <Container>
+        <PageWrapper>
+          <H1>Downloads</H1>
+          <Content>
+            <Description><P>Download some goodies to start designing your decentralized apps. This is an ever growing collection of components for Sketch and Figma, and interactive components for Framer X.</P><P>If you would like to contribute to designing new components, please get in touch! The best way is to open an issue on <A href={"https://github.com/lorikeetui/design/issues"}>https://github.com/lorikeetui/design/issues</A></P></Description>
+            <DownloadWrapper>
+              <DownloadLink src={SketchSVG} text={"Download for Sketch"} href={"#"}/>
+              <DownloadLink src={figmaSVG} text={"Download for Figma"} href={"#"}/>
+              <DownloadLink src={FramerSVG} text={"Download for Framer X"} href={"#"}/>
+            </DownloadWrapper>
+          </Content>
+        </PageWrapper>
+      </Container>
     </Section>
   </Page>
 )
 
+const DownloadLink = (props) =>
```

Comment:
We can separate the two components to be on different files. I would put DownloadLink inside src/components/Downloads

Reply:
ok! I moved them to src/components/DownloadLink, looks cleaner :)